src/models/layout/inspection.rs

```diff
@@ -0,0 +1,7 @@
+//! in-toto layout's Inspection
+
+use serde_derive::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Serialize, Deserialize, PartialEq)]
+pub struct Inspection {}
+// TODO Inspection
```

Comment:
I wonder, is the reason you left this as a TODO is becuase you wanted to implement the runlib bits of inspections in a separate PR?

Reply:
Yea, I just want to get ideas from reviews and carry on to implement `Inspection` after the weekend

Reply:
cool! 

I think I'm happy with having inspections after the fact on a separate PR to keep things tractable.